docs/backends/jaeger.mdx

```diff
@@ -1,34 +1,30 @@
 ---
-title: "Jaeger"
+title: 'Jaeger'
 ---
 
 ## Configuring Jaeger Backend
 
 Version v1.35 of Jaeger introduced the ability to receive OpenTelemetry trace data via the OpenTelemetry Protocol (OTLP).
 This allows to create a new Jaeger backend by simply specifying the Jaeger OTLP gRPC unencrypted URL.
 
-The endpoint format is `host:port`. 
-- host is required
-- port is optional and defaults to the default OTLP gRPC port `4317`.
+- **JAEGER_URL** - Endpoint, the format is `host:port`.
```

Comment:
I wonder if we should use the code name here `JAEGER_URL` of the display name

Reply:
Key name makes most sense to me, it's also self explanatory.